lib/miq_expression/field.rb

```diff
@@ -21,6 +21,17 @@ def self.is_field?(field)
     parse(field)&.valid? || false
   end
 
+  def column_values
+    return [] unless valid?
+
```

Comment:
This could end up returning a huge amount of data. Would it be possible to limit the data coming back? Perhaps we can push the `.uniq` down to the DB and use `select` to return only the column? Also, it may make sense to force a limit on the number of rows returned to protect against running this query on a table with millions of rows.

I'm not sure what our options are for virtual attributes.

Reply:
👍 yes that's right, I rewrote it to use SQL and for uniqueness I am using distinct.

Then I added limit to 100 rows. (maybe we will update this once we will do other progress on new expression editor)

and for now I removed virtual attributes because it can cause performance issues.

thanks